lib/octocatalog-diff/catalog-util/command.rb

```diff
@@ -95,9 +107,16 @@ def setup
           --no-daemonize
           --no-ca
```

Comment:
Check on puppet 6.19 and `--no-ca` option is removed now

Reply:
Good catch. I've added a conditional to only include the `--no-ca` option on Puppet versions < 6.0. Thanks!